app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,44 @@
+// TODO
+// 1. Read Collection - GET /tasks [OK]
+// 2. Read Item - GET /tasks/{id}
+// 3. Create - POST /tasks/{id}
+// 4. Update - PUT/PATCH /tasks/{id}
+// 5. Delete - DELETE /tasks/{id}
+
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("tasks")
+@CrossOrigin
+public class TaskController {
+    private List<Task> tasks = new ArrayList<>();
+    private Long newId = 0L;
+
+    @RequestMapping(path = "", method = RequestMethod.GET)
+    public List<Task> list() {
+        return tasks;
+    }
```

Comment:
`path = ""`는 사용하지 않아도 괜찮을 것 같아요 

```suggestion
    @GetMapping
    public List<Task> list() {
        return tasks;
    }
```

Reply:
오..!! 깔끔하니다